lib/glue.rb

```diff
@@ -78,16 +78,30 @@ def issue_on_clipboard(issue)
 
     html = "<a href='#{link}'>#{issue.key}</a>: #{summary}#{impact_html}"
     text = "#{issue.key}: #{summary}"
-    
     slack_text = "#{issue.key}: #{summary}\n#{link}"
-
     Clipboard.insert!(html, text, slack_text)
 
     # @notifier.show_message!("Added #{issue.key} to clipboard")
-    
     [summary, link, impact]
   end
-  
+
+  def build_branch_name_from_issue(issue, copy_to_clipboard = true)
+    issue or return
+
+    summary, = @jira.issue_description_and_link_from_issue(issue)
+
+    # Formatting: downcase -> replace specific non-alphanumeric characters with spaces -> remove non-alphanumberic characters -> replace spaces with join
+    summary_formatted = summary.downcase.tr(':;()-.', ' ').gsub(/[^a-z0-9\s&]/, '').split(' ').join('_')
+    month             = Date.today.month
+    month_formatted   = month > 9 ? month : "0#{month}"
+    date              = "#{Date.today.year % 100}#{month_formatted}"
```

Comment:
the month and date formatting code can all be simplified to:

```
Time.new.strftime("%y%m")
```

And per other comment, I'd like this code to first check if there is a config setting for the prefix, and otherwise using this as default).  I would make the config value just be a date format string, e.g. `"%y%m"` or `"%Y-%m"` etc.. anything that can be fed into `strftime`.

https://ruby-doc.org/core-2.4.0/Time.html#method-i-strftime